package.json

```diff
@@ -9,6 +9,7 @@
     "build": "nest build",
     "format": "prettier --write \"src/**/*.ts\" \"test/**/*.ts\"",
     "start": "nest start",
+    "dev": "nest start --watch",
```

Comment:
Duplicated command, there is `yarn start:dev` with same subcommand already.

Reply:
Right, it was duplicated on purpose while testing turborepo setup. Possibly I can just have it removed and update the compose file command to point to the `start:dev` task.